hexa/pipelines/models.py

```diff
@@ -199,6 +199,7 @@ class Meta:
     memory_request = models.CharField(blank=True, max_length=32)
     memory_limit = models.CharField(blank=True, max_length=32)
     recipients = models.ManyToManyField(User, through="PipelineRecipient")
+    webhook_token = models.UUIDField(null=True, blank=True)
```

Comment:
Your initial solution was good. You have to use a TimestampSigner for this BUT you still have to support the case where it's an UUID during the migration phase (that can last 2 weeks)